src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    public Manufacturer create(Manufacturer manufacturer);
+
+    public Optional<Manufacturer> get(Long id);

```

Comment:
methods in interface are public abstract by default
```suggestion
    Optional<Manufacturer> get(Long id);
```